packages/components/src/components/form/Select/index.tsx

```diff
@@ -240,45 +238,19 @@ const Select = ({
     useKeyPressScroll,
     isSearchable = false,
     minWidth = 'initial',
-    borderWidth = 2,
-    borderRadius = 4,
     state,
+    'data-test': dataTest,
     ...props
-}: Props) => {
-    const selectRef: React.RefObject<ReactSelect<Option>> | null | undefined = useRef(null);
+}: SelectProps) => {
+    const selectRef: React.RefObject<ReactSelect<Option>> = useRef(null);
```

Comment:
will be better to use built in useRef generic aka `useRef<ReactSelect<Option>>(null)`

Reply:
100%

done